app/assets/stylesheets/deploys.scss

```diff
@@ -106,3 +106,14 @@ td.status {
 .deployment-alert-popover {
   max-width: 340px;
 }
+
+@-webkit-keyframes myfirst
+{
+  0%   { background: $samsonDark; color: $info; }
+  50%  { background: $info; color: $samsonDark; }
+  100% { background: $samsonDark; color: $info; }
+}
+
+.badge-deploys {
+  -webkit-animation: myfirst 6s infinite;
```

Comment:
My super personal opinion would be `-webkit-` plus unprefixed. Firefox works with unprefixed.